packages/btc/src/api/sendRbf.ts

```diff
@@ -0,0 +1,169 @@
+  if (props.changeIndex !== undefined) {
+    const changeOutput = outputs[props.changeIndex];
+    const isReturnableOutput = changeOutput && 'address' in changeOutput;
+    const changeOutputAddress = isReturnableOutput ? changeOutput.address : undefined;
+    if (!changeOutput) {
+      throw TxBuildError.withComment(ErrorCodes.INVALID_CHANGE_OUTPUT, `outputs[${props.changeIndex}] is not found`);
+    }
+    if (!isReturnableOutput) {
+      throw TxBuildError.withComment(
+        ErrorCodes.INVALID_CHANGE_OUTPUT,
+        `outputs[${props.changeIndex}] is not a returnable output for change`,
+      );
+    }
+    if (changeOutputAddress && changeAddress && changeAddress !== changeOutputAddress) {
+      throw TxBuildError.withComment(
+        ErrorCodes.INVALID_CHANGE_OUTPUT,
+        `outputs[${props.changeIndex}].address does not match the specified changeAddress`,
+      );
+    }
+    if (changeOutputAddress && !changeAddress) {
+      changeAddress = changeOutputAddress;
+    }
+
+    const config = networkTypeToConfig(props.source.networkType);
+    const minUtxoSatoshi = props.minUtxoSatoshi ?? config.btcUtxoDustLimit;
+    changeOutput.minUtxoSatoshi = minUtxoSatoshi;
+    changeOutput.value = minUtxoSatoshi;
+    changeOutput.protected = true;
+    changeOutput.fixed = false;
+  }
+
+  // Fee rate
+  const requireGreaterFeeAndRate = props.requireGreaterFeeAndRate ?? true;
+  let feeRate: number | undefined = props.feeRate;
+  if (requireGreaterFeeAndRate && !feeRate) {
+    const feeRates = await props.source.service.getBtcRecommendedFeeRates();
+    feeRate = feeRates.fastestFee;
+  }
+
+  // The RBF transaction should offer a higher fee rate
+  const previousInsValue = inputs.reduce((sum, input) => sum + input.value, 0);
+  const previousOutsValue = previousTx.outs.reduce((sum, output) => sum + output.value, 0);
+  const previousFee = previousInsValue - previousOutsValue;
+  const previousFeeRate = Math.floor(previousFee / previousTx.virtualSize());
+  if (requireGreaterFeeAndRate && feeRate !== undefined && feeRate <= previousFeeRate) {
+    throw TxBuildError.withComment(
+      ErrorCodes.INVALID_FEE_RATE,
+      `RBF should offer a higher fee rate, previous: ${previousFeeRate}, current: ${feeRate}`,
+    );
+  }
+
+  // Exclude the previous transaction's outputs during the collection
+  const previousTxId = previousTx.getId();
+  const excludeUtxos: BaseOutput[] = previousTx.outs.map((_, index) => ({
+    txid: previousTxId,
+    vout: index,
+  }));
+
+  // Build RBF transaction
+  const res = await createSendUtxosBuilder({
+    inputs,
+    outputs,
+    excludeUtxos,
+    changeAddress,
+    from: props.from,
+    source: props.source,
+    feeRate: props.feeRate,
+    fromPubkey: props.fromPubkey,
+    minUtxoSatoshi: props.minUtxoSatoshi,
+    onlyConfirmedUtxos: props.onlyConfirmedUtxos,
+    skipInputsValidation: true,
+  });
+
+  // The RBF transaction should offer a higher fee amount
```

Comment:
WHen onlyConfirmedUtxos=false, there is a known issue: https://github.com/ckb-cell/rgbpp-sdk/pull/150#issuecomment-2139325384

Reply:
For the sendRbf() API specifically, which method do you think is better: 
1. Add a default value for the onlyConfirmedUtxos
2. Remove the onlyConfirmedUtxos prop from the API

Reply:
Added a default value `true` to the onlyConfirmedutxos prop at: [627d833](https://github.com/ckb-cell/rgbpp-sdk/pull/150/commits/627d8330f46ca7cc677bf5309a69a7f35e82a68f)

Reply:
LGTM